bin/dartdoc.dart

```diff
@@ -76,22 +88,24 @@ void _printUsageAndExit(ArgParser parser) {
 
 ArgParser _createArgsParser() {
   var parser = new ArgParser();
-  parser.addOption('exclude',
-      help: 'Comma-separated list of library names to ignore.');
-  parser.addOption('dart-sdk',
-      help: "Location of the Dart SDK. Use if SDK isn't automatically located.");
-  parser.addOption('hosted-url',
-      help: 'URL where the docs will be hosted (used to generate the sitemap).');
   parser.addFlag('help',
       abbr: 'h', negatable: false, help: 'Show command help.');
   parser.addFlag('version',
       help: 'Display the version for $NAME.', negatable: false);
+  parser.addOption('dart-sdk',
+      help: "Location of the Dart SDK. Use if SDK isn't automatically located.");
   parser.addFlag('sdk-docs', help: 'Generate ONLY the docs for the Dart SDK.');
   parser.addOption('sdk-readme',
       help: 'Path to the SDK description file. Use if generating Dart SDK docs.');
-  parser.addOption('footer',
-      help: 'Path to an HTML, inserted into the footer of every page.');
+  parser.addOption('input-dir',
+      help: 'Path to source directory, defaults to current directory');
   parser.addOption('output',
       help: 'Path to output directory.', defaultsTo: 'docs');
+  parser.addOption('footer',
+      help: 'HTML text, inserted into the footer of every page.');
```

Comment:
Sorry, I was not clear. Right now you have to pass the HTML text in on command line. This is not ideal, and will be changed to pass in path to a file with the contents.